src/Schema/CMS/Events/BatchEditFlow.ts

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
Hiya!

I've inferred some of the data below from your [tracking doc](https://www.figma.com/design/14GMdTIbcn725fyvPU02XL/Bulk-Edits-Beta?node-id=5649-64263&t=iwZGMY8RGpe8wcrN-0) @leodmz 

Please feedback welcome!